aip/general/0009.md

```diff
@@ -102,6 +102,22 @@ The version of an API or a group of APIs if they are defined
 together. An API Version is often represented by a string, such as "v1", and
 presents in API requests and Protocol Buffers package names.
 
+### Client
+
+Code that calls the API or operates on the resource data at rest. These clients
+may be a program that performs a specific task on an API, or be a generic tool
```

Comment:
1. The phrase, "These clients may be a program", does not have subject verb agreement. 
1. You're speaking about "clients", so saying, "these" leads one to think "which?". 
1. In, "on an API", "on" is not quite the right proposition, at least in my opinion: I would be explicit and say, "by calling". 
1. This is a run-on sentence. You need to remove the comma because the phrase after it could not stand alone as a complete sentence.
1. "or be a generic tool" is passive voice, I think, but just generally sounds like incorrect English.
1. I'm pretty sure that the reader knows that CLI is command line interface.

Recommend,

> Clients are programs that perform a specific tasks by calling an API or generic tools, such as CLIs, that expose the API in a user-accessible fashion.

Reply:
ack, I went with your suggestion, but added a clause on "resource data at rest".